var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -59,9 +59,6 @@ class Gcc(AutotoolsPackage):
     variant('binutils',
             default=sys.platform != 'darwin',
             description="Build via binutils")
-    variant('gold',
-            default=sys.platform != 'darwin',
-            description="Build the gold linker plugin for ld-based LTO")
```

Comment:
I removed this variant because it doesn't belong here. If you want to build binutils with `+gold`, you should run:
```
$ spack install gcc ^binutils+gold
```
Variants of dependencies don't belong in packages higher up in the DAG if they don't affect that package.